docarray/proto/docarray.proto

```diff
@@ -120,6 +120,9 @@ message DocumentProto {
   // Evaluations performed on the document, each element corresponds to a metric
   map<string, NamedScoreProto> evaluations = 19;
 
+  // system-defined meta attributes represented in a structured data value.
+  google.protobuf.Struct _metadata = 20;
```

Comment:
Please also be aware that the integer values in the `google.protobufStruct` will be cast to float values automatically when serialize/deserialize to/from protobuf.  As shown in current document tags:

```
from docarray import Document

d = Document(tags={'x': 1})

pb = d.to_protobuf(d)
d = Document.from_protobuf(pb)
print(d.tags)
```
returns
```
{'x': 1.0}
```

Reply:
thanks for pointing this out. Actually I didn't include tests for protobuf serialization/deserialization, will do so soon